includes/preprocessor-symbols.md

```diff
@@ -1,21 +1,5 @@
- ```
-.NET Framework 2.0        --> NET20
-.NET Framework 3.5        --> NET35
-.NET Framework 4.0        --> NET40
-.NET Framework 4.5        --> NET45
-.NET Framework 4.5.1      --> NET451
-.NET Framework 4.5.2      --> NET452
-.NET Framework 4.6        --> NET46
-.NET Framework 4.6.1      --> NET461
-.NET Framework 4.6.2      --> NET462
-.NET Framework 4.7        --> NET47
-.NET Standard 1.0         --> NETSTANDARD1_0
-.NET Standard 1.1         --> NETSTANDARD1_1
-.NET Standard 1.2         --> NETSTANDARD1_2
-.NET Standard 1.3         --> NETSTANDARD1_3
-.NET Standard 1.4         --> NETSTANDARD1_4
-.NET Standard 1.5         --> NETSTANDARD1_5
-.NET Standard 1.6         --> NETSTANDARD1_6
-.NET Core Application 1.0 --> NETCOREAPP1_0
-.NET Core Application 1.1 --> NETCOREAPP1_1
-```
+| Target Frameworks          | Symbols |
+| -------------------------- | ------- |
+| .NET Framework 2.0-4.7.1   | `NET20`, `NET35`, `NET40`, `NET45`, `NET451`, `NET452`, `NET46`, `NET461`, `NET462`, `NET47`, `NET471` |
```

Comment:
I like the new format. Should we just call it .NET Framework, .NET Standard and .NET Core without version or application on the first column?